doc/source/whatsnew/v1.0.0.rst

```diff
@@ -31,7 +31,7 @@ Other enhancements
 
 .. _whatsnew_1000.api_breaking:
 
--
+- :meth:`MultiIndex.from_product` infers level names from inputs if possible (:issue:`27292`)
```

Comment:
But this above the `.. _whatsnew_1000.api_breaking:` line.

And maybe say "when not explicitly provided" rather than "if possible".

Reply:
> Put this above the `.. _whatsnew_1000.api_breaking:` line.

It looks like a few of the section reference names are slightly misaligned with ticks throughout the whatsnew:
```rst
Enhancements
~~~~~~~~~~~~

.. _whatsnew_1000.enhancements.other:

-
-

Other enhancements
^^^^^^^^^^^^^^^^^^

.. _whatsnew_1000.api_breaking:

-
-

Backwards incompatible API changes
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

.. _whatsnew_1000.api.other:

- :class:`pandas.core.groupby.GroupBy.transform` now raises on invalid operation names (:issue:`27489`).
```
I'll open up a quick PR to fix it across the board.  Might result in a merge conflict here but should be straightforward to resolve.

Reply:
I made the change to the text in whatsnew, holding off on changing anything else if jschendel is going to be changing other parts of the file.

Reply:
I just merged my changes to the whatsnew, so you should be good to make the additional changes now.  Should just need to address the merge conflict by making sure your entry is above the `.. _whatsnew_1000.api_breaking:` line.